source/_posts/2015-03-26-creating-a-dynamic-single-page-app-for-our-genome-team-using-react.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: "Creating a dynamic single page app for our Genome Team using React"
```

Comment:
Title ideas:
* Trying out React
* React and Rails
* Meerkat snapchat bitcoin I dunno